model/model.go

```diff
@@ -0,0 +1,130 @@
+// Package model defines object and payload models
+package model
+
+const (
+	// BaseURL is the definition of ovalfi base url
+	BaseURL = "https://sandbox-api.ovalfi-app.com"
+	// Signature sample sandbox environment signature
+	Signature = "segsalerty@gmail.com"
+	// BearerToken sample sandbox environment bearer token
+	BearerToken = "segun"
```

Comment:
Not sure what's going on here

Reply:
@odetolakehinde

Reply:
rename to some random string for now.
and how have you tested this?

Reply:
I created examples in the `example.go` file to make API calls to these endpoints. I've tested all of them and they work